pkg/monitoring/metrics.go

```diff
@@ -60,41 +60,41 @@ func GetRecordRulesDesc(namespace string) []RecordRulesDesc {
 	return []RecordRulesDesc{
 		{
 			MetricOpts{
-				"kubevirt_cdi_operator_up_total",
+				"kubevirt_cdi_operator_up",
```

Comment:
@akalenyu @avlitman This metric's value can be different than 0/1, right?
(Expr: sum(up{namespace='%s', pod=~'cdi-operator-.*'} or vector(0)))
If so, I think kubevirt_cdi_operator_up may be a bit misleading, as it sounds like a boolean metric.

Reply:
@assafad Agreed to leave as is.